Modular Scripts/File Monitoring/configure-file-monitoring.sh

```diff
@@ -405,7 +405,7 @@ write21ConfFileContents() {
             \$ActionSendStreamDriverPermittedPeer *.loggly.com
 
             #RsyslogGnuTLS
-            \$DefaultNetstreamDriverCAFile /etc/rsyslog.d/keys/ca.d/logs-01.loggly.com_sha12.crt
+            \$DefaultNetstreamDriverCAFile $CA_FILE_PATH
```

Comment:
Where is $CA_FILE_PATH defined?

Reply:
`CA_FILE_PATH` (as well as other methods and vars) is defined in [configure-linux.sh](https://github.com/loggly/install-script/blob/e960ee810b1c6dcb1f5333dfa906b40fe70ffff4/Linux%20Script/configure-linux.sh#L506) script, which is dependency for [configure-file-monitoring.sh](https://github.com/loggly/install-script/blob/e960ee810b1c6dcb1f5333dfa906b40fe70ffff4/Modular%20Scripts/File%20Monitoring/configure-file-monitoring.sh#L6).